common/templates/student/edx_ace/passwordreset/email/body.html

```diff
@@ -6,33 +6,55 @@
-                <br />
-            </p>
-
             {% if failed %}
+                <h1>
+                    {% trans "Password Reset Unsuccessful" %}
+                </h1>
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}You have received this email message because this email address was submitted to {{ platform_name }} for password assistance.{% endblocktrans %}
+                    <br />
+                </p>
+
                 <p style="color: rgba(0,0,0,.75);">
-                    {% blocktrans %}However, there is currently no user account associated with your email address: {{ email_address }}.{% endblocktrans %}
+                    {% blocktrans %}This email address is not associated with an account on {{ platform_name }}. As a result, {{ platform_name }} has taken no action and will not use or share this email address.{% endblocktrans %}
                     <br />
                 </p>
 
                 <p style="color: rgba(0,0,0,.75);">
-                    {% trans "If you did not request this change, you can ignore this email." %}
+                    {% blocktrans %}<strong>To reset your {{ platform_name }} account password</strong>, go to the {{ platform_name }} sign-in page, click the <strong>Forgot Password?</strong> link, and enter the email address that is associated with your {{ platform_name }} account.{% endblocktrans %}
                     <br />
                 </p>
+
+                {% if help_link %}
+                    <p style="color: rgba(0,0,0,.75);">
+                            {% blocktrans %}For more information, see this {{ platform_name }} <a href="{{ help_link }}">Help Center article</a>.{% endblocktrans %}
+                        <br />
+                    </p>
+                {% endif %}
+
             {% else %}
+                <h1>
+                    {% trans "Password Reset" %}
+                </h1>
+
                 <p style="color: rgba(0,0,0,.75);">
-                    {% trans "If you didn't request this change, you can disregard this email - we have not yet reset your password." %}
+                    {% blocktrans %}{{platform_name}} has received a request to reset the password for this account.{% endblocktrans %}
+                    <br />
+                </p>
+
+                <p style="color: rgba(0,0,0,.75);">
+                    {% trans "If you requested a password reset, click the following link, or paste the URL in your browser's address bar." %}
                     <br />
                 </p>
 
                 {% trans "Change my Password" as course_cta_text %}
 
                 {% include "ace_common/edx_ace/common/return_to_course_cta.html" with course_cta_text=course_cta_text course_cta_url=reset_link %}
+
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}If you did not request a password reset, you can ignore this email. platform_name has taken no action.{% endblocktrans %}
```

Comment:
Is "platform_name" here missing some braces?

Reply:
It certainly is!  Nice catch